### PR TITLE
Picks 281 implement schedule scraper cron job script

### PR DESCRIPTION
**Pull request by mightys**

### CODE REVIEW

**Comment by the-freshlord** [comment]
Needs rebase

File: common/models/schedule.json [hunk 1]
```diff
@@ -1,11 +1,45 @@
...
+    },
+    "year": {
+      "type": "string",
+      "required": true
+    },
+    "week": {
+      "type": "string",
+      "required": true
+    },
+    "homeTeamName": {
```
Comment (the-freshlord) [inline common/models/schedule.json:32]: Any reason for the home team name?

File: server/boot/09-nfl-scrapper.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+            console.log('error creating schedule', error);
+          });
+        });
+      });
+    } else {
+      console.log('week and year out of bound', week, year);
+    }
+  });
+  console.log('Finished boot script 9');
+};
```
Comment (the-freshlord) [inline server/boot/09-nfl-scrapper.js:151]: We need to also scrape passing yards and rush yards for each team for future games

**Review by the-freshlord — CHANGES_REQUESTED** [review]
Needs rebase

File: server/boot/09-nfl-scrapper.js (hunk 2 above)
Comment (mightys) [reply server/boot/09-nfl-scrapper.js:151]: yeah, I'm adding that in another task/script

File: common/models/schedule.json (hunk 1 above)
Comment (mightys) [reply common/models/schedule.json:32]: yes, for easy access to append other information gotten from another script/scrapper such as the weather information and rush yard etc.

File: server/boot/09-nfl-scrapper.js (hunk 2 above)
Comment (the-freshlord) [reply server/boot/09-nfl-scrapper.js:151]: Needs another rebase

**Review by the-freshlord — APPROVED** [review]

**Pull request merged**